.gitignore

```diff
@@ -4,8 +4,6 @@
 /node_modules
 /.pnp
 .pnp.js
-package-lock.json
```

Comment:
please avoid modify this file... these files do not have to be uploaded to the repository